decidim/decidim
# Redesign: make the actions buttons consistent

#### :tophat: What? Why?

This PR tries to fix the consistency in the Actions buttons for the components. Most notably:

1. Makes the bulk "Action" button in the cases of Proposals and Budgets to behave as other dropdown buttons (same colors, with the chevron aka arrow in the right)
2. Adds the chevron arrow in other dropdowns (like Import and Export all) 
3. Fixes a problem with the translations and the different resources. 
4. Fixes a consistency order problem between the different buttons
 
##### About the translations and the different resources
 
Seems like it's overkill to not use the resource name, until you see  that having the generic brings too many bugs:

1. Some buttons say things like "New Project" instead of "New project"
2. For fixing that, on some buttons we were doing the equivalent to "New #{"Project".downcase}". This approach does not work on some languages where the subject needs to be capitalized (like German)
3. The generic approach meant that in some languages with gender, we had things like "Nuevo propuesta" (mixing genders) and leaving a bad impression on our translators when they didn't even have the fault :/

So, the fix is to actually separate the "New" button translation for each resource so translators can understand better and translate it in the correct gender and casing.
 
##### About the New Order of buttons

Right now we have the different bulk/actions buttons for the components without a consistent order. This commit tries to fix that.

The New Order (:D) is the following:

1. "Actions" (with dropdown) in the cases that there is one (like Proposals and Budgets)
2. "Export all" (with dropdown)
3. "Import" (with dropdown)
4. Other random actions (like "Send voting reminders" in Budgets)
5. New resource (like New budget). This one should be always the first, counting from the right side.

#### Testing

1. Sign in as admin
2. Go to the different components and see the different actions buttons in the components

### :camera: Screenshots

![Screenshot of the budgets component](https://github.com/decidim/decidim/assets/717367/1b0efc11-273f-4044-8f05-e3e96b4fcdc5)


:hearts: Thank you!
